.github/knip.ts

```diff
@@ -1,11 +1,11 @@
 import type { KnipConfig } from "knip";
 
 const config: KnipConfig = {
-  entry: ["build/index.ts"],
+  entry: ["src/worker.ts"],
   project: ["src/**/*.ts"],
-  ignore: ["src/types/config.ts"],
+  ignoreBinaries: ["build", "i", "format:cspell", "awk", "lsof"],
```

Comment:
Out of curiosity, what is "i"?

Reply:
@FernandVEYRIER I was also surprised. This comes from `bun i` which is a short form of `bun install`.